src/main/kotlin/com/github/avrokotlin/avro4k/annotations.kt

```diff
@@ -8,6 +8,10 @@ import kotlinx.serialization.SerialInfo
 @Target(AnnotationTarget.PROPERTY, AnnotationTarget.CLASS)
 annotation class AvroProp(val key: String, val value: String)
 
+@SerialInfo
+@Target(AnnotationTarget.PROPERTY, AnnotationTarget.CLASS)
+annotation class AvroProps(val key: String, vararg val value: String)
```

Comment:
I think to improve the readability, it should be named `values` as it can contains one or more values, and it should be an array to distinguish key from values as shown in the example:
```kotlin
  @Serializable
  @AvroProps("cold", ["play", "and", "more"])
  data class TypeAnnotated(val str: String)
```
```suggestion
annotation class AvroProps(val key: String, vararg val values: Array<String>)
```